client/state/themes/schema.js

```diff
@@ -0,0 +1,92 @@
+const themeSchema = {
+	title: 'Theme',
+	type: 'object',
+	properties: {
+		id: {
+			type: 'string'
+		},
```

Comment:
May be more concise if these were on a single lines:
```
id: { type: 'string' },
name: { type: 'string' },
...
```
Unless the linter won't have it.